pkgs/tools/virtualization/ec2instanceconnectcli/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, boto3 }:
+
+buildPythonPackage rec {
+  pname = "ec2instanceconnectcli";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-VaCyCnEhSx1I3bNo57p0IXf92+tO1tT7KSUXzO1IyIU=";
+  };
+
+  propagatedBuildInputs = [ boto3 ];
+  
+  # has no tests
+  doCheck = false;
+  
+  pythonImportsCheck = [ "ec2instanceconnectcli" ];
+
+  meta = with lib; {
+    description = "Command Line Interface for AWS EC2 Instance Connect";
+    homepage = "https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Connect-using-EC2-Instance-Connect.html";
```

Comment:
Another viable option here would behttps://github.com/aws/aws-ec2-instance-connect-cli

Reply:
I think that would be better.

Reply:
Updated and rebased.